erniebot-agent/erniebot_agent/tools/base.py

```diff
@@ -124,6 +116,28 @@ async def parse_file_from_response(
         )
         return local_file
 
+    # 2. parse file from file_mimetypes
+    if len(file_mimetypes) > 1:
+        raise RemoteToolError(
+            "There are multi file-mimetypes defined in response schema, "
+            "it only support one file mime-type in response.",
```

Comment:
1. NotImpelmentedError
2. Multiple file MIME types are defined in the Response Schema. Currently, only single file output is supported. Please ensure that only one file MIME type is defined in the Response Schema.